README.md

```diff
@@ -42,12 +42,12 @@ print(mlc.Dataset(url).metadata.to_json())
 import tensorflow_datasets as tfds
 builder = tfds.core.dataset_builders.CroissantBuilder(
     jsonld=url,
-    record_set_ids=["record_set_fashion_mnist"],
+    record_set_ids=["fashion_mnist"],
     file_format='array_record',
 )
 builder.download_and_prepare()
 # 4. Split for training/testing
-train, test = builder.as_data_source(split=['default[:80%]', 'default[80%:]'])
+train, test = builder.as_data_source(split=['train[:80%]', 'test[80%:]'])
```

Comment:
the 80% split was, IIRC, just to make up for the missing split definiton in the early croissant definitions for huggingface dataset. Can't you just `train, test = builder.as_data_source(split=['train', 'test'])` ?